playbooks/byo/openshift-cluster/upgrades/docker/docker_upgrade.yml

```diff
@@ -23,11 +23,23 @@
   serial: 1
   any_errors_fatal: true
   tasks:
-  - name: Prepare for Node draining
-    command: >
-      {{ openshift.common.client_binary }} adm manage-node {{ openshift.node.nodename }} --schedulable=false
+  - name: Load lib_openshift modules
+    include_role:
+      name: lib_openshift
```

Comment:
@ashcrow and @ingvagabund, keep this in mind for your refactoring.  If you have a play, with only a tasks: section, you can add a roles: section to include lib_openshift.